crates/cheatcodes/src/evm.rs

```diff
@@ -224,6 +227,28 @@ impl Cheatcode for resumeGasMeteringCall {
     }
 }
 
+impl Cheatcode for lastGasUsedCall {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self {} = self;
+        ensure!(
+            state.config.isolate,
+            "`lastGasUsed` is only available in isolated mode (`--isolate`)"
+        );
+        if let Some(latest_gas_usage) = &state.latest_gas_usage {
+            Ok(latest_gas_usage.abi_encode())
+        } else {
+            Ok(GasRecord {
+                gasLimit: 0,
+                gasTotalUsed: 0,
+                gasMemoryUsed: 0,
+                gasRefunded: 0,
+                gasRemaining: 0,
+            }
```

Comment:
Currently the `Gas` struct in `vm.rs` does not have a default, hence this workaround. Would be preferable if this was handled

Reply:
Could be replaced with an ensure instead of returning a default case, allows us to remove the duplicate struct and is more clear to users @zerosnacks